packages/ui5-config/scripts/download-ui5yaml-schema.js

```diff
@@ -0,0 +1,17 @@
+const axios = require('axios');
+const fs = require('fs');
+
+const schemaURL = 'https://raw.githubusercontent.com/SAP/ui5-tooling/gh-pages/schema/ui5.yaml.json';
+
+/**
+ * Downloads the UI5 yaml schema from the official repository and saves it to the dist folder.
+ */
+async function downloadUI5YamlSchema() {
+    console.info('Downloading UI5 yaml schema...');
```

Comment:
```suggestion
    console.info('Downloading UI5 YAML schema...');
```

Reply:
Done in 604b3e90f6b1a742735db764bc0f38e1cb28a5e4